frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -215,6 +216,7 @@ class AccountsSummary extends Component<Props, State> {
                     </Header>
                     <div className="innerContainer scrollableContainer">
                         <div className="content padded">
+                            <AddBuyOnEmptyBalances balances={balances} />
```

Comment:
Small issue:
I ran into an edgecase, while testing where the accounts are not fully loaded but the CTA quickly flickers shows and then disappears, tested with a few empty and non empty LTC and BTC accounts. We have syncStatus in the state but I am not sure if that is helpful.

As accounts are there, but balances can be undefined and then get populated one by one, I think the condition to render this component could be something like this `accounts.length === balances && Object.keys(balances).length`

Reply:
@BrianCraig can you try to reprodce?
(maybe reconfigure accounts or delete appfolder.dev to reproduce)

Could you try to only render the component on the summary page if `accounts.length === balances && Object.keys(balances).length` or similar.

Reply:
![Screenshot from 2021-12-09 08-03-04](https://user-images.githubusercontent.com/4956754/145385131-caf2e362-a12b-46e3-804e-a65ae1d18667.png)
![Screenshot from 2021-12-09 08-03-24](https://user-images.githubusercontent.com/4956754/145385137-5f5fe72d-7517-4093-989a-29e7ae4f0085.png)
![Screenshot from 2021-12-09 08-04-25](https://user-images.githubusercontent.com/4956754/145385139-54590ffa-a3c2-41b4-9689-6f94314c97ea.png)
![Screenshot from 2021-12-09 08-04-46](https://user-images.githubusercontent.com/4956754/145385140-a08fc2bf-eba7-4fe7-a3a1-1da8a4e7e643.png)

Reply:
The issue only appears during balance / account loading. And the CTA quickly appears but then as soon as there is a balance it disapears, sometimes jus a flickr, sometimes for a few seconds. depends on how fast the accounts/balance load

Reply:
:+1: I'll have to ckeck with the profilig tools to see what states have the balances and acounts

Reply:
I chnged to show only when accounts and balances matcches quantity as you said :+1: